src/AutoBuild.jl

```diff
@@ -161,7 +161,8 @@ function build_tarballs(ARGS, src_name, src_version, sources, script,
     # If the user passed in a platform (or a few, comma-separated) on the
     # command-line, use that instead of our default platforms
     if length(ARGS) > 0
-        platforms = platform_key_abi.(split(ARGS[1], ","))
+        _platform_key_abi(p::AbstractString) = p == "any" ? AnyPlatform() : platform_key_abi(p)
+        platforms = _platform_key_abi.(split(ARGS[1], ","))
```

Comment:
Ah, this is necessary because of how we build on Yggdrasil.  yeah, okay.

Reply:
:heavy_check_mark: